### PR TITLE
LCI-A 10 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/legions_chant.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Legion's Chant
+ManaCost:2 W
+Types:Sorcery Chorus
+K:Starting intensity:3
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | WithTotalCMC$ X | ChangeNum$ Y | Hidden$ True | AnyNumber$ True | ChangeType$ Creature.YouOwn | SubAbility$ DBIntensify | SpellDescription$ Return any number of creature cards with total mana value X or less from your graveyard to the battlefield, where X is CARDNAME's Intensity.
+SVar:DBIntensify:DB$ Intensify | AllDefined$ Card.YouOwn+Chorus | DefinedDesc$ all Chorus cards you own | SpellDescription$ All Chorus cards you own intensify by 1.
+SVar:Y:Count$ValidGraveyard Card.YouOwn
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/legions_chant.txt:7]: fix type

File: forge-gui/res/cardsfolder/upcoming/propagator_primordium.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Propagator Primordium
+ManaCost:1 G
+Types:Creature Fungus
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigConjure | TriggerDescription$ When CARDNAME enters the battlefield, conjure two cards named Propagator Primordium into your graveyard.
+SVar:TrigConjure:DB$ MakeCard |  Amount$ 2 | Conjure$ True | Name$ Propagator Primordium | Zone$ Graveyard
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ At the beginning of your upkeep, put a spore counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ SPORE | CounterNum$ 1
+A:AB$ ChangeZone | Cost$ SubCounter<3/SPORE> | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Creature.Fungus+YouOwn | TgtPrompt$ Choose target Fungus creaure card in your graveyard | SubAbility$ DBToken | SpellDescription$ Return target Fungus creature card from your graveyard to the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/propagator_primordium.txt:9]: fix sub

File: forge-gui/res/cardsfolder/upcoming/kamachal_ships_mascot.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Types:Legendary Creature Dinosoaur Pirate
+PT:2/2
+K:Menace
+A:AB$ Pump | Cost$ R | NumAtt$ +1 | SpellDescription$ CARDNAME gets +1/+0 until end of turn.
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | Execute$ TrigTreasure | CombatDamage$ True | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, create a Treasure token. Then exile from that player's library a random card with mana value equal to the damage dealt. You may cast that card this turn.
+SVar:TrigTreasure:DB$ Token | TokenScript$ c_a_treasure_sac | SubAbility$ DBChoose
+SVar:DBChoose:DB$ ChooseCard | Choices$ Card.cmcEQX+ControlledBy TriggeredTarget | ChoiceZone$ Library | Amount$ 1 | AtRandom$ True | SubAbility$ DBChangeZoneAll
+SVar:DBChangeZoneAll:DB$ ChangeZoneAll | Origin$ Library | Destination$ Exile | ChangeType$ Card.ChosenCard | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | StaticAbilities$ MayPlay | RememberObjects$ Remembered | SubAbility$ DBCleanup
+SVar:MayPlay:Mode$ Continuous | Affected$ Card.IsRemembered | EffectZone$ Command | AffectedZone$ Exile | MayPlay$ True | Description$ You may play that card this turn
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kamachal_ships_mascot.txt:12]: play != cast

File: forge-gui/res/cardsfolder/upcoming/ribald_shanty.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:Ribald Shanty
+ManaCost:1 R
+Types:Instant Chorus
+K:Starting intensity:2
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker  | TgtPrompt$ Select target creature or planeswalker | NumDmg$ X | SubAbility$ DBIntensify | SpellDescription$ This spell deals damage equal to its intensity to target creature or planeswalker.
+SVar:DBIntensify:DB$ Intensify | AllDefined$ Card.YouOwn+Chorus | DefinedDesc$ all Chorus cards you own | SpellDescription$ All Chorus cards you own intensify by 1.
+SVar:X:Count$Intensity
+DeckHints:Type$Chorus
+Oracle:Starting intensity 2\nThis spell deals damage equal to its intensity to target creature or planeswalker.\nAll Chorus cards you own intensify by 1.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ribald_shanty.txt:9]: add whitespace line

File: forge-gui/res/cardsfolder/upcoming/ribald_shanty.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Ribald Shanty
+ManaCost:1 R
+Types:Instant Chorus
+K:Starting intensity:2
+A:SP$ DealDamage | ValidTgts$ Creature,Planeswalker  | TgtPrompt$ Select target creature or planeswalker | NumDmg$ X | SubAbility$ DBIntensify | SpellDescription$ This spell deals damage equal to its intensity to target creature or planeswalker.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ribald_shanty.txt:5]: spacing

**Comment by Northmoc** [comment]
Don't forget to make sure all your .txt files have a whitespace line at the end for style 😉

**Comment by Northmoc** [comment]
> Don't forget to make sure all your .txt files have a whitespace line at the end for style 😉

there is probably some programming reason as well 😅

**Review by tool4ever — APPROVED** [review]

**Pull request merged**